libs/application/templates/children-residence-change/src/lib/ChildrenResidenceChangeTemplate.ts

```diff
@@ -13,7 +13,7 @@ import { dataSchema } from './dataSchema'
 
 type Events = { type: DefaultEvents.ASSIGN } | { type: DefaultEvents.SUBMIT }
 
-enum States {
+export enum States {
```

Comment:
Wondering if we wan't to have a type file for the types we use in the template or if we wan't to export them directly like this when we need them.

Reply:
I would say it just depends..One thing regarding this though is that I would probably change this enum to `ApplicationStates` and then we can get rid of `{ States as ApplicationStates }` where the type is being imported.